pkg/service/rest_client_test.go

```diff
@@ -0,0 +1,24 @@
+package service
+
+import (
+	"testing"
+
+	"kusionstack.io/kclvm-go/pkg/spec/gpyrpc"
+)
+
+func TestCallRestMethod_ping(t *testing.T) {
```

Comment:
Add an error test case like method not found.

Reply:
fixed